src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
```

Comment:
- [ ] Also Please make this file a class-based component, as the requirement says.

Reply:
Oh, yes. Thank you @dasileker

Reply:
This issue is well resolved! ✔️